src/Kiota.Builder/Writers/Python/CodeMethodWriter.cs

```diff
@@ -289,7 +289,7 @@ private static void WriteQueryParametersMapper(CodeMethod codeElement, CodeClass
         writer.WriteLine($"{errorMappingVarName}: Dict[str, ParsableFactory] = {{");
         writer.IncreaseIndent();
         foreach(var errorMapping in codeElement.ErrorMappings) {
-            writer.WriteLine($"\"{errorMapping.Key.ToUpperInvariant()}\": o_data_error.{errorMapping.Value.Name}.get_from_discriminator_value(),");
+            writer.WriteLine($"\"{errorMapping.Key.ToUpperInvariant()}\": o_data_error.{errorMapping.Value.Name},");
```

Comment:
Commenting a bit late here but I'm worried that this hardcoded namespace name will throw out of balance generation for clients that are not for odata based APIs?